.github/workflows/_run-app-as-provider-contract-tests.yml

```diff
@@ -0,0 +1,68 @@
+# This workflow is meant to be run on a pact provider build. It will check pacts
```

Comment:
This is same as `_run-provider-contract-tests.yml` but with a different name. `_run-provider-contract-tests.yml` will be removed when it is no longer used.